ai/marketplace/plugins/my/skills/project-claude-setup/specs/2026-05-11-coexistence-implementation.md

```
@@ -0,0 +1,1132 @@
+
+- [ ] **Step 1: Re-read current SKILL.md**
+
+```bash
+sed -n '25,45p' ~/.dotfiles/ai/marketplace/plugins/my/skills/project-claude-setup/SKILL.md
+```
+Confirm Prereq #5 is the abort-on-tracked-CLAUDE.md gate that's being replaced.
+
+- [ ] **Step 2: Replace Prereq #5**
+
+Current (lines 31–39):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)